procrastinate/contrib/django/admin.py

```diff
@@ -128,3 +148,7 @@ def summary(self, instance: models.ProcrastinateJob) -> str:
                 ).strip()
             )
         return ""
+
+    def full_args_view(self, request, job_id):
+        instance = models.ProcrastinateJob.objects.get(id=job_id)
+        return JsonResponse(instance.args)
```

Comment:
The base idea is nice ! Though I'm not sure the link is the most ergonomic choice. It certainly is the easiest to implement.

How about a small js script that would replace the args with the full version, or copy to the clipboard ? (note: due to Content Security Policy, we can't be sure JS code will execute, so if we go this way we have to make sure the no-js case is working)

That can be left for a subsequent PR though.

Reply:
This wasn't a quick hack but I actually prefer to just get a super low-level "raw" `application/json` response.

It's usually the most performant with large args, and also allows the use of browser extensions that render a collapsible tree etc.

Reply:
Fine by me.